x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/actions/trial_license_complete_tier/check_privileges.ts

```diff
@@ -85,7 +85,7 @@ export default ({ getService }: FtrProviderContext) => {
 
           // TODO: https://github.com/elastic/kibana/pull/121644 clean up, make type-safe
           expect(body?.execution_summary?.last_execution.message).to.contain(
-            `This rule may not have the required read privileges to the following index patterns: ["${index[0]}"]`
+            `The concrete indices that match this index pattern may not be available to this rule's api key: ["${index[0]}"]`
```

Comment:
```suggestion
            `All concrete indices that match this index pattern may not be available to this rule's api key: ["${index[0]}"]. Only indices available to rule's api key have been searched. See [Detections Authorization](https://www.elastic.co/docs/api/doc/elasticsearch/v8/operation/operation-security-get-api-key) for details.`
```

Thoughts on that?

cc @approksiu

Reply:
Chatting with @approksiu - she found a better doc link - https://www.elastic.co/guide/en/kibana/current/alerting-setup.html#alerting-authorization

One thing to check is whether the links need to point to different versions for 8.18/9.0 or if we're able to just link to latest.

Reply:
Hiay! The Docs team is trying to avoid hard-coded links in the UI and are asking teams to use the [Kibana docs links service](https://docs.elastic.dev/docs/kibana-doc-links) instead. It looks like that page has some instructions for handling 8.x and 9.x doc links, so maybe that'll help to answer your question Yara.